.github/workflows/black.yml

```diff
@@ -0,0 +1,12 @@
+  
+name: Black Formatter
+on: [push, pull_request]
+jobs:
+  black-format:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Black Code Formatter
+        uses: lgeiger/black-action@master
+        with:
+          args: "."
```

Comment:
This should be `args: ". --check"`, see https://github.com/lgeiger/black-action

Reply:
If you want to automatically "black" PRs, then you would need something like https://github.com/cclauss/autoblack

Reply:
@ocaisa it seems like this may be more troublesome than it looks. The regression test failed when running black manually.